apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/Transaction.java

```diff
@@ -109,6 +132,8 @@
 
     byte[] getBytes();
 
+    byte[] bytes();
```

Comment:
bytes() and getBytes() seems to be the same

Reply:
Not exactly...
In TransactionImpl.java
**byte[] getBytes();** returns COPY of **byte[] bytes();**

Reply:
Do we really require bytes() in interface?

Reply:
Seems like this method should be private

Reply:
> Do we really require bytes() in interface?

I can remove it from interface declaration and make 'casting' when needed.